test/e2e/cmd/run/run.go

```diff
@@ -149,6 +149,7 @@ func (h *helper) initTestContext() error {
 		TestRun:               h.testRunName,
 		TestTimeout:           h.testTimeout,
 		IgnoreWebhookFailures: h.ignoreWebhookFailures,
+		OcpCluster:            h.kubectl("get", "clusterversion") == nil,
```

Comment:
My goal with this was to try to not make OCP a special case. I do realize that the end result here is that in this case I'm doing just that.

My hope, however, is for this to not stay as is forever. Hopefully, all the ifs/cases required for OCP will go away eventually as they are mostly related to how restricted OCP is by default and not because of some technical limitation.

This is all to say that I'd love for this special case to be an internal thing rather than it being exposed externally so that we can remove freely when we reach that ideal state.

Curious to hear your thoughts.

Reply:
Got it, I'm fine with keeping this as is.